### ISSUE TITLE
ESelect выкидывает ошибку при выборе опции

### ISSUE THREAD

**Issue by Neirokin** [egal/widget-library#151]
После обновления виджета для установки дефолтного значения теперь передаётся объект, а не массив. Из-за этого при выбор опции происходит ошибка
![image](https://user-images.githubusercontent.com/56441777/182351098-854c9561-9b3d-4b60-a238-82b57dd49267.png)

**Comment by novkri** [comment]
Да, но можно было просто передавать объект в массиве и тогда не надо было бы писать отдельную функцию на @update:modelValue для каждого селекта и доставать нулевой элемент чтобы обновить modelValue извне (сейчас в 35 версии этот эмит все равно будет возвращать объект в массиве после выбора другого значения, даже если изначально передан просто объект. А так селект бы сам перезаписал modelValue )
